Makefile

```diff
@@ -0,0 +1,53 @@
+export GO111MODULE=on
+export GOPROXY=https://goproxy.io
```

Comment:
What's the goal of this proxy?

Reply:
Speed up download dependencies. Works on go mod

Reply:
Do you think we could consider switching to https://proxy.golang.org?